torch/_functorch/partitioners.py

```diff
@@ -749,11 +741,10 @@ def classify_nodes(joint_module):
         method_to_operator(m)
         for m in magic_methods
     ]
-
     recomputable_ops = set(recomputable_ops) if recomputable_ops is not None else set(default_recomputable_ops)
 
     random_ops = [aten.native_dropout, aten.rand_like, aten.randn_like]
-    compute_intensive_ops = [aten.mm, aten.convolution, aten.convolution_backward, aten.bmm, aten.addmm, aten.upsample_bilinear2d, aten._softmax, aten._softmax_backward_data, aten.native_layer_norm, aten.native_layer_norm_backward, aten.native_batch_norm, aten.native_batch_norm_backward, aten._native_batch_norm_legit, aten._batch_norm_with_update, aten.batch_norm_backward]  # noqa: E501,B950
+    compute_intensive_ops = [aten.mm, aten.convolution, aten.convolution_backward, aten.bmm, aten.addmm, aten._scaled_dot_product_flash_attention, aten._scaled_dot_product_efficient_attention, aten.upsample_bilinear2d]  # noqa: E501,B950
```

Comment:
batchnorm/softmax/layernorm were able to be removed bc of reduction changes ? how did you decide upsample_bilinear2. 

i think you might want max_pool2d_backward as well

Reply:
They were removed since we never see them nowadays at this point.

This list is not used unless aggressive recompute is turned on. In that case, I think it might make more sense to not recompute things like upsampling. This is more preparation for later changes for more user control for recomputation.